### PR TITLE
Refactor logstash pipelines to improve CPU untilization

### PR DESCRIPTION
**Pull request by maismail**
https://hopsworks.atlassian.net/browse/HOPSWORKS-3222

### CODE REVIEW

File: templates/default/services.conf.erb [hunk 1]
```diff
@@ -19,24 +19,50 @@ filter {
       match => {"message" => "\[#\|%{NOTSPACE:templogdate}\|%{LOGLEVEL:priority}\|%{DATA:product_id}\|%{DATA:logger_name}\|%{GREEDYDATA:key_value_pairs}\|%{GREEDYDATA:log_message}\|#\]"}
     }
   } else if [service] == "mysql" {
-    grok {
+    dissect {
       # datetime [string] severity -- message
       # 2021-08-12 09:16:52
-      match => {"message" => "%{DATA:templogdate} \[%{DATA:logger_name}\] %{LOGLEVEL:priority}((.|\r|\n)*)-- %{GREEDYDATA:log_message}"}
+      mapping => {"message" => "%{templogdate} [%{logger_name}] %{priority}     -- %{log_message}"}
```
Comment (gibchikafa) [inline templates/default/services.conf.erb:25]: This is leaving space at the end of the` priority`. Maybe this:` %{templogdate} %{+templogdate} [%{logger_name}] %{priority} %{?blank}--%{}%{log_message}`. I am using this https://dissect-tester.jorgelbg.me/ to test

**Comment by maismail** [comment]
> Looks good. We are only printing errors, which makes sense to not overload. What happens if a service goes crazy printing errors, will we be ok?

I tried killing the RonDB datanode on my cluster to check the load on Logstash and it went up to 250% CPU due to service logs overload (also filebeat in this situation would go to 200-300%) but it quickly return back to 10% once the database is up and running again

File: templates/default/services.conf.erb (hunk 1 above)
Comment (maismail) [reply templates/default/services.conf.erb:25]: It doesn't if you keep the spaces count to 5, but probably better to use suggestion instead

Comment (maismail) [reply templates/default/services.conf.erb]: @SirOibaf btw, this pattern doesn't apply for all the logs under mysql/cluster, for now i kept the same old behaviour

**Pull request merged**